tests/test_feature_selection.py

```diff
@@ -0,0 +1,101 @@
+# Authors: Soledad Galli <solegalli1@gmail.com>
+# License: BSD 3 clause
+import pytest
```

Comment:
please remove lines with my name and email. You are creating the code. So no need to have my name on it :)